ui/src/components/Terminal/index.vue

```diff
@@ -311,15 +311,15 @@ onMounted(() => {
       message.error('WebSocket connection is not established');
       return;
     }
-    const data = shareLinkRequest.data || {};
+    const data = shareLinkRequest.data.requestData || {};
     const perm = data.action_permission || data.action_perm || 'writable';
     socket.value.send(
       formatMessage(
         terminalId.value,
         FORMATTER_MESSAGE_TYPE.TERMINAL_SHARE,
         JSON.stringify({
           origin: origin,
-          session: sessionId.value,
+          session: shareLinkRequest.data.sessionId,
           users: data.users,
           expired_time: data.expired_time,
           action_permission: perm
```

Comment:
There doesn't seem to be any errors or irregularities identified in the provided code snippet. It appears to be a simple websocket event handling logic with some basic data manipulations, but it would still benefit from additional error handling and validation checks where appropriate.